pontoon/base/models.py

```diff
@@ -561,16 +561,33 @@ def get_chart_data(stats):
     )
 
 
-def get_locales_with_stats(project):
+def get_locales_with_stats():
+    """Add chart data to locales."""
+
+    locales = Locale.objects.filter(stats__isnull=False).distinct()
+
+    for locale in locales:
+        stats = Stats.objects.filter(
+            resource__entity__obsolete=False,
+            locale=locale).distinct()
+
+        locale.chart = get_chart_data(stats)
```

Comment:
I agree, and the same goes for get_projects_with_stats(). We should make these changes at the same time, because they follow the same logic. They affects 7 differents views (locale, locales, project, projects, admin, translate, search), so they should be done separately.